src/navigator-card-body.jsx

```diff
@@ -370,7 +370,7 @@ export const NavigatorCardBody = ({
     }
 };
 
-function Item({ file, selected, setSelected, onDoubleClickNavigate, isGrid }) {
+const Item = React.memo(function Item({ file, isSelected, setSelected, onDoubleClickNavigate, isGrid }) {
```

Comment:
I would prefer a comment here in the code, as it was not super obvious. But also fine to merge without.

Reply:
Done!